<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sort</class>
 <widget class="QDialog" name="Sort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sort</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>11</y>
     <width>80</width>
     <height>54</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="moreButton">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>117</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>More</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="secondGroupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>114</y>
     <width>173</width>
     <height>97</height>
    </rect>
   </property>
   <property name="title">
    <string>Second key</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>31</y>
      <width>43</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Column</string>
    </property>
   </widget>
   <widget class="QComboBox" name="secondColumnBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>31</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Пусто</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>61</y>
      <width>52</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort order</string>
    </property>
   </widget>
   <widget class="QComboBox" name="secondOrderBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>61</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ascending</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Descending</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="thirdGroupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>217</y>
     <width>173</width>
     <height>97</height>
    </rect>
   </property>
   <property name="title">
    <string>Third key</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Column</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="thirdColumnBox">
      <item>
       <property name="text">
        <string>Пусто</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Sort order</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="thirdOrderBox">
      <item>
       <property name="text">
        <string>Ascending</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Descending</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="firstGroupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>173</width>
     <height>97</height>
    </rect>
   </property>
   <property name="title">
    <string>First key</string>
   </property>
   <widget class="QComboBox" name="firstOrderBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>61</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ascending</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Descending</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>61</y>
      <width>52</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort order</string>
    </property>
   </widget>
   <widget class="QComboBox" name="firstColumnBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>31</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Пусто</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>31</y>
      <width>43</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Column</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>firstColumnBox</tabstop>
  <tabstop>firstOrderBox</tabstop>
  <tabstop>moreButton</tabstop>
  <tabstop>secondColumnBox</tabstop>
  <tabstop>secondOrderBox</tabstop>
  <tabstop>thirdColumnBox</tabstop>
  <tabstop>thirdOrderBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Sort</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Sort</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>secondGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>thirdGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
